python/chronos/src/bigdl/chronos/detector/anomaly/ae_detector.py

```diff
@@ -160,19 +161,17 @@ def fit(self, y):
             ae_model = create_torch_model(self.compress_rate, len(y[0]))
             optimizer = optim.Adadelta(ae_model.parameters(), lr=self.lr)
             criterion = nn.BCELoss()
+            internal_model = Trainer.compile(model=ae_model, loss=criterion,
+                                             optimizer=optimizer)
+            trainer = Trainer(logger=False, max_epochs=self.epochs)
```

Comment:
try if num_processes>1 helps the training performance. if helps, please follow this policy: https://github.com/intel-analytics/BigDL/blob/main/python/chronos/src/bigdl/chronos/forecaster/lstm_forecaster.py#L132-L135

Reply:
checkpoint to false

Reply:
if checkpoint is false, distributed_backend should be spawn.